web/styles/app_variables.css

```diff
@@ -767,6 +771,10 @@
     --color-message-formatting-controls-container: hsl(232deg 30% 96%);
     --color-message-content-container-border: hsl(0deg 0% 0% / 10%);
     --color-message-content-container-border-focus: hsl(0deg 0% 57%);
+    --color-message-content-container-border-over-limit: hsl(0deg 76% 65%);
```

Comment:
We already define this color below in dark theme block, so why do we again need this here?
Infact we should remove the color defined below, if both light and dark theme have same colors, and define this here because we do not show any border currently in light theme if the message content length crosses the limit.

Reply:
It is mentioned in the issue description by @alya as you can see below:

![image](https://github.com/user-attachments/assets/0d08b5c8-f3ea-44b8-b1e5-2cf64aed3ae5)

But it is not specifically mentioned whether it is only for dark theme or both.